examples/python/wasm-in-pytket.py

```diff
@@ -0,0 +1,136 @@
+# pip install pytket~=1.13.0
+
+# # WASM function calls with pytket
+# The WASM module in python allows you to add external classical functions from a compiled web assembly (WASM) to the circuit.
```

Comment:
Maybe there's some motivation that could be added here? Someone reading this tutorial might wonder why using WASM alongside quantum circuits is useful